### PR TITLE
docs: update Tailwind link

### LINKED ISSUE #3553: Tailwind link goes to GitHub, doesn't make sense
## Possible issue

On this page: https://docs.astro.build/en/guides/styling/, a couple of the Tailwind links go to: https://github.com/withastro/astro/tree/main/packages/integrations/tailwind

## Motivation

It doesn't make sense to me to refer a user to the raw Github integration. We do have this documented under: https://docs.astro.build/en/guides/integrations-guide/tailwind/

## Screenshot:

Screenshot 2023-06-22 at 22 34 57

## Possible solution

Replace the link with: https://docs.astro.build/en/guides/integrations-guide/tailwind/

### PR DESCRIPTION
**Pull request by ElianCodes**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)

#### Description

- replaces link to raw GitHub page to documented Tailwind Integration page
- Closes #3553

### CODE REVIEW

File: src/content/docs/en/guides/styling.mdx [hunk 1]
```diff
@@ -429,9 +429,22 @@ import "../components/make-it-purple.css"
 
 ## CSS Integrations
 
-Astro comes with support for adding popular CSS libraries, tools and frameworks to your project like [Tailwind][tailwind] and more!
+Astro comes with support for adding popular CSS libraries, tools and frameworks to your project like Tailwind and more!
```
Comment (ElianCodes) [inline src/content/docs/en/guides/styling.mdx:432]: I guess adding in the official Tailwind site here, couldn't hurt
```suggestion
Astro comes with support for adding popular CSS libraries, tools and frameworks to your project like [Tailwind](https://tailwindcss.com) and more!
```

File: src/content/docs/en/guides/styling.mdx [hunk 2]
```diff
@@ -429,9 +429,22 @@ import "../components/make-it-purple.css"
...
+
+To use Taiwind in your project, install the offical [Astro Tailwind integration][tailwind] using the `astro add` command for your package manager:
+
+```sh
+# Using NPM
+npx astro add tailwind
+# Using Yarn
+yarn astro add tailwind
+# Using PNPM
+pnpm astro add tailwind
```
Comment (ElianCodes) [inline src/content/docs/en/guides/styling.mdx:444]: Don't we wanne make this the package manager tabs components?

**Review by ElianCodes — COMMENTED** [review]
Thanks @sarah11918! I still left a couple suggestions I want your opinion on. Afterwards I'll merge

File: src/content/docs/en/guides/styling.mdx (hunk 1 above)
Comment (sarah11918) [reply src/content/docs/en/guides/styling.mdx:432]: Agreed! Like it!

File: src/content/docs/en/guides/styling.mdx (hunk 2 above)
Comment (ElianCodes) [reply src/content/docs/en/guides/styling.mdx:444]: Allright! I'll make a mental not to revisit this soon!

**Pull request merged**